### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by dchess**
Depends on #2 

Adds basic logging and email notifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,4 +35,23 @@ DB_SERVER=
 DB_USER=
 DB_PWD=
 DB_SCHEMA=
+
+# Email Notifications
+ENABLE_MAILER=1
```
Comment (susanxiong) [inline README.md:40]: I like these variable names. They're clearer than our old ones.

File: main.py [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+from mailer import Mailer
+
+
+def configure_logging():
+    logging.basicConfig(
+        handlers=[
+            logging.FileHandler(filename="data/app.log", mode="w+"),
+            logging.StreamHandler(sys.stdout),
+        ],
+        level=logging.DEBUG,
```
Comment (susanxiong) [inline main.py:16]: Will this stay at DEBUG or change to INFO when deployed?

**Review by susanxiong — APPROVED** [review]
Can you add docstrings to the mailer?

**Review by zkagin — APPROVED** [review]
Looks good to me.

**Pull request merged**